src/Quiz.js

```diff
@@ -294,6 +293,48 @@ const Quiz = () => {
       setScore(player.score);
       setPower(player.item);
     }
+    updateEnemies(userDataArray);
```

Comment:
maybe this could be better this way tho : 
if (showModal)
{
 updateEnemies(userDataArray);
}